### ISSUE TITLE
crash on upload of non-image file

### ISSUE THREAD

**Issue by theotheroracle** [Nheko-Reborn/nheko#1139]
images seem to work fine but any other type  of file freezes, and then crashes nheko ( 1.10.0 ) it also seems to freeze my shell which gets killed by -9 presumably by oom ( memory leak ? )

( you can see cpu and memory spike when i attempt to upload a file )
![Screenshot from 2022-08-02 17-17-56](https://user-images.githubusercontent.com/20211219/182435279-73d16e0a-acb8-4276-a2d9-ef20df80404e.png)

in the console i get the warning `[2022-08-02 17:14:11.987] [qml] [warning] qrc:/qml/UploadBox.qml:47:17: QML Image: Binding loop detected for property "sourceSize.height" (qrc:/qml/UploadBox.qml:47, )`

i have nvidia graphics if that makes a difference at all

**Comment by deepbluev7** [comment]
How big was the file you wanted to send? Nheko tries to thumbnail video and audio files, for which it tries to play them in the background.

**Comment by theotheroracle** [comment]
small text file

**Comment by deepbluev7** [comment]
Hm, I haven't seen that so far... any specific file ending?

**Comment by theotheroracle** [comment]
happens with a lot of file endings, txt, md, html, mov, basically anything that's not a photo . i tried running it with gdb but gdb gets oom'd, unsure how else to record a backtrace

**Comment by deepbluev7** [comment]
How did you install Nheko?

**Comment by theotheroracle** [comment]
through dnf, and i installed debug symbols

**Comment by deepbluev7** [comment]
Can you show a crash log then? Or is it just killed because of oom?

**Comment by theotheroracle** [comment]
it's just oom

**Comment by theotheroracle** [comment]
i tried running with valgrind but it wouldn't run because it thought qt wasn't installed ? so idk what to do abot that

**Comment by fschwa** [comment]
Same here. Goes bonkers with CPU and MEM when selecting a text file and then gets killed after a minute or two.

This is v0.10.2 on Debian Sid.

**Comment by fschwa** [comment]
valgrind output


Clicking the Add File dialog:
```
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x13287719: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x41C5A7F9: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x42188EC9: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x4210BF29: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x41D68E19: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x41F575E9: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x41F58FA9: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x41D6C259: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x41D6D149: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0xD74AA99: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2CE25: ???
==13514==    by 0x4239F677: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2CE25: ???
==13514==    by 0x38F769D7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2CE25: ???
==13514==    by 0x402078D7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2CFC5: ???
==13514==    by 0x424A0D87: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x40D21699: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x41111809: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x42136839: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x36AA3039: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x395D20C9: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x3B0EBEA9: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x3B211249: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x3C053739: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x4031A639: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x367E1E79: ???
==13514== 
==13565== 
==13565== HEAP SUMMARY:
==13565==     in use at exit: 69,149,534 bytes in 514,390 blocks
==13565==   total heap usage: 2,298,259 allocs, 1,783,869 frees, 425,006,230 bytes allocated
==13565== 
==13565== LEAK SUMMARY:
==13565==    definitely lost: 780,176 bytes in 1,068 blocks
==13565==    indirectly lost: 645,562 bytes in 21,533 blocks
==13565==      possibly lost: 5,619,066 bytes in 49,379 blocks
==13565==    still reachable: 61,959,626 bytes in 441,798 blocks
==13565==                       of which reachable via heuristic:
==13565==                         newarray           : 845,608 bytes in 1,661 blocks
==13565==                         multipleinheritance: 277,704 bytes in 382 blocks
==13565==         suppressed: 0 bytes in 0 blocks
==13565== Rerun with --leak-check=full to see details of leaked memory
==13565== 
==13565== Use --track-origins=yes to see where uninitialised values come from
==13565== For lists of detected and suppressed errors, rerun with: -s
==13565== ERROR SUMMARY: 169 errors from 80 contexts (suppressed: 0 from 0)
[2022-10-09 19:56:31.830] [qml] [warning] The desktop entry file "/usr/share/applications/kcm_krunnersettings.desktop" has Type= "Application" but no Exec line (:0, )
[2022-10-09 19:56:31.832] [qml] [warning] Invalid Service :  "/usr/share/applications/kcm_krunnersettings.desktop" (:0, )
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x41551F19: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x3C0BE979: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x410C3DD9: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x419ADC79: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x41C099B9: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x4178E439: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D2D965: ???
==13514==    by 0x22456849: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29275: ???
==13514==    by 0x401A32E7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29275: ???
==13514==    by 0x40481FD7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29275: ???
==13514==    by 0x40303E87: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29275: ???
==13514==    by 0x41750807: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29275: ???
==13514==    by 0x4070C297: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29275: ???
==13514==    by 0x40B71C17: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29275: ???
==13514==    by 0x41068717: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29275: ???
==13514==    by 0x4106A1A7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29275: ???
==13514==    by 0x418371B7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29275: ???
==13514==    by 0x42B9EF67: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29275: ???
==13514==    by 0x42BA0157: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29275: ???
==13514==    by 0x42BC1967: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29275: ???
==13514==    by 0x376DC7A7: ???
==13514== 
==13514== 
==13514== More than 100 errors detected.  Subsequent errors
==13514== will still be recorded, but in less detail than before.
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29275: ???
==13514==    by 0x391573E7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29275: ???
==13514==    by 0x3B26B047: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29275: ???
==13514==    by 0x3969D207: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29275: ???
==13514==    by 0x36D03C07: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29275: ???
==13514==    by 0x3B5FDD87: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0xD8CF537: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x4122B677: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x2226CDA7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x41982B47: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x36F28C97: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x414B60F7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x40316757: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x4239D0D7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x4100EE67: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x43465A37: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x41CE5F67: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x42A5BDD7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x40590167: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x41CD3AF7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x41B074F7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29420: ???
==13514==    by 0x41B074F7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x42BDE4B7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x40196967: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x221A4D17: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0xE4CFF97: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x23A1FD17: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x3CC0B927: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x3719E037: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x40C42E07: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0xD99EFA7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x411051D7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0xDFA1AD7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x41E43C27: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0xE6DCA97: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x36FFFE77: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x36849B97: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x38F7CD87: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x4027C707: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x39857F97: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x41EB1847: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x3AA247C7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x3B14DC67: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x3B08CAA7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x3942C047: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x3CDA6817: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x40CBF247: ???
==13514== 
==13578== 
==13578== HEAP SUMMARY:
==13578==     in use at exit: 69,894,343 bytes in 521,168 blocks
==13578==   total heap usage: 3,100,903 allocs, 2,579,735 frees, 541,339,919 bytes allocated
==13578== 
==13578== LEAK SUMMARY:
==13578==    definitely lost: 793,558 bytes in 1,093 blocks
==13578==    indirectly lost: 647,400 bytes in 21,562 blocks
==13578==      possibly lost: 5,623,498 bytes in 49,372 blocks
==13578==    still reachable: 62,684,783 bytes in 448,529 blocks
==13578==                       of which reachable via heuristic:
==13578==                         newarray           : 845,728 bytes in 1,664 blocks
==13578==                         multipleinheritance: 280,624 bytes in 396 blocks
==13578==         suppressed: 0 bytes in 0 blocks
==13578== Rerun with --leak-check=full to see details of leaked memory
==13578== 
==13578== Use --track-origins=yes to see where uninitialised values come from
==13578== For lists of detected and suppressed errors, rerun with: -s
==13578== ERROR SUMMARY: 235 errors from 146 contexts (suppressed: 0 from 0)
==13591== 
==13591== HEAP SUMMARY:
==13591==     in use at exit: 69,100,424 bytes in 511,066 blocks
==13591==   total heap usage: 3,137,745 allocs, 2,626,679 frees, 548,389,235 bytes allocated
==13591== 
==13591== LEAK SUMMARY:
==13591==    definitely lost: 791,552 bytes in 1,090 blocks
==13591==    indirectly lost: 646,970 bytes in 21,563 blocks
==13591==      possibly lost: 4,921,054 bytes in 39,179 blocks
==13591==    still reachable: 62,595,744 bytes in 448,622 blocks
==13591==                       of which reachable via heuristic:
==13591==                         newarray           : 845,728 bytes in 1,664 blocks
==13591==                         multipleinheritance: 283,224 bytes in 396 blocks
==13591==         suppressed: 0 bytes in 0 blocks
==13591== Rerun with --leak-check=full to see details of leaked memory
==13591== 
==13591== Use --track-origins=yes to see where uninitialised values come from
==13591== For lists of detected and suppressed errors, rerun with: -s
==13591== ERROR SUMMARY: 235 errors from 146 contexts (suppressed: 0 from 0)
```

After selecting the file
```
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x40660BE7: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x42B4BE07: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D295E3: ???
==13514==    by 0xE6C9A27: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29415: ???
==13514==    by 0x36D0A637: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29C9D: ???
==13514==    by 0xDC41577: ???
==13514== 
==13514== Conditional jump or move depends on uninitialised value(s)
==13514==    at 0x22D29DA5: ???
==13514==    by 0x3BF4CB27: ???
==13514== 
[2022-10-09 19:57:06.842] [qml] [warning] qrc:/qml/UploadBox.qml:47:17: QML Image: Binding loop detected for property "sourceSize.height" (qrc:/qml/UploadBox.qml:47, )
==13514== Warning: set address range perms: large range [0x859c8b040, 0x86e89bb50) (undefined)
==13514== Warning: set address range perms: large range [0x86e89c040, 0x8834acb50) (undefined)
==13514== Warning: set address range perms: large range [0x8834ad040, 0x8980bdb50) (undefined)
==13514== Warning: set address range perms: large range [0x86e89c028, 0x8834acb68) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b028, 0x86e89bb68) (noaccess)
==13514== Warning: set address range perms: large range [0x8834ad028, 0x8980bdb68) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b040, 0x86e89bb50) (undefined)
==13514== Warning: set address range perms: large range [0x86e89c040, 0x8834acb50) (undefined)
==13514== Warning: set address range perms: large range [0x8834ad040, 0x8980bdb50) (undefined)
==13514== Warning: set address range perms: large range [0x86e89c028, 0x8834acb68) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b028, 0x86e89bb68) (noaccess)
==13514== Warning: set address range perms: large range [0x8834ad028, 0x8980bdb68) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b040, 0x87a3707c4) (undefined)
==13514== Warning: set address range perms: large range [0x87a371040, 0x89aa567c4) (undefined)
==13514== Warning: set address range perms: large range [0x89aa57040, 0x8bb13c7c4) (undefined)
==13514== Warning: set address range perms: large range [0x87a371028, 0x89aa567dc) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b028, 0x87a3707dc) (noaccess)
==13514== Warning: set address range perms: large range [0x89aa57028, 0x8bb13c7dc) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b040, 0x87a3707c4) (undefined)
==13514== Warning: set address range perms: large range [0x87a371040, 0x89aa567c4) (undefined)
==13514== Warning: set address range perms: large range [0x89aa57040, 0x8bb13c7c4) (undefined)
==13514== Warning: set address range perms: large range [0x87a371028, 0x89aa567dc) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b028, 0x87a3707dc) (noaccess)
==13514== Warning: set address range perms: large range [0x89aa57028, 0x8bb13c7dc) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b040, 0x88c775280) (undefined)
==13514== Warning: set address range perms: large range [0x88c776040, 0x8bf260280) (undefined)
==13514== Warning: set address range perms: large range [0x8bf261040, 0x8f1d4b280) (undefined)
==13514== Warning: set address range perms: large range [0x88c776028, 0x8bf260298) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b028, 0x88c775298) (noaccess)
==13514== Warning: set address range perms: large range [0x8bf261028, 0x8f1d4b298) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b040, 0x88c775280) (undefined)
==13514== Warning: set address range perms: large range [0x88c776040, 0x8bf260280) (undefined)
==13514== Warning: set address range perms: large range [0x8bf261040, 0x8f1d4b280) (undefined)
==13514== Warning: set address range perms: large range [0x88c776028, 0x8bf260298) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b028, 0x88c775298) (noaccess)
==13514== Warning: set address range perms: large range [0x8bf261028, 0x8f1d4b298) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b040, 0x8a8f98dc4) (undefined)
==13514== Warning: set address range perms: large range [0x8a8f99040, 0x8f82a6dc4) (undefined)
==13514== Warning: set address range perms: large range [0x8f82a7040, 0x9475b4dc4) (undefined)
==13514== Warning: set address range perms: large range [0x8a8f99028, 0x8f82a6ddc) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b028, 0x8a8f98ddc) (noaccess)
==13514== Warning: set address range perms: large range [0x8f82a7028, 0x9475b4ddc) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b040, 0x8a8f98dc4) (undefined)
==13514== Warning: set address range perms: large range [0x8a8f99040, 0x8f82a6dc4) (undefined)
==13514== Warning: set address range perms: large range [0x8f82a7040, 0x9475b4dc4) (undefined)
==13514== Warning: set address range perms: large range [0x8a8f99028, 0x8f82a6ddc) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b028, 0x8a8f98ddc) (noaccess)
==13514== Warning: set address range perms: large range [0x8f82a7028, 0x9475b4ddc) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b040, 0x8d5845864) (undefined)
==13514== Warning: set address range perms: large range [0x8d5846040, 0x951400864) (undefined)
==13514== Warning: set address range perms: large range [0x951401040, 0x9ccfbb864) (undefined)
==13514== Warning: set address range perms: large range [0x8d5846028, 0x95140087c) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b028, 0x8d584587c) (noaccess)
==13514== Warning: set address range perms: large range [0x951401028, 0x9ccfbb87c) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b040, 0x8d5845864) (undefined)
==13514== Warning: set address range perms: large range [0x8d5846040, 0x951400864) (undefined)
==13514== Warning: set address range perms: large range [0x951401040, 0x9ccfbb864) (undefined)
==13514== Warning: set address range perms: large range [0x8d5846028, 0x95140087c) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b028, 0x8d584587c) (noaccess)
==13514== Warning: set address range perms: large range [0x951401028, 0x9ccfbb87c) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b040, 0x91b1d0880) (undefined)
==13514== Warning: set address range perms: large range [0x91b1d1040, 0x9dc716880) (undefined)
==13514== Warning: set address range perms: large range [0x9dc717040, 0xa9dc5c880) (undefined)
==13514== Warning: set address range perms: large range [0x91b1d1028, 0x9dc716898) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b028, 0x91b1d0898) (noaccess)
==13514== Warning: set address range perms: large range [0x859c8b040, 0x91b1d0880) (undefined)
==13514== Warning: set address range perms: large range [0x91b1d1040, 0x9dc716880) (undefined)
==13514== Warning: set address range perms: large range [0xa9dc5d040, 0xb5f1a2880) (undefined)
Killed
```

**Comment by deepbluev7** [comment]
Should be fixed by #1215

**Comment by LorenDB** [comment]
This indeed seems to be fixed.
